src/main/java/org/mitre/bonnie/cqlTranslationServer/Main.java

```diff
@@ -44,9 +45,12 @@ public static HttpServer startServer() {
    */
   public static void main(String[] args) throws IOException {
     final HttpServer server = startServer();
-    System.out.println(String.format("Jersey app started with WADL available at "
-            + "%sapplication.wadl\nHit enter to stop it...", BASE_URI));
-    System.in.read();
-    server.shutdownNow();
+
```

Comment:
Sounds goods, though it has to be off by default as it causes the process to crash when a production container is demonized. Looks like the current v1.0.2 tags are both working as expected.